screenapiexample/src/main/java/net/gini/android/vision/screen/AnalysisActivity.java

```diff
@@ -61,17 +57,26 @@ public void onClick(View v) {
                     }
                 });
             }
+
+            @Override
+            public void onExtractionsReceived(Map<String, SpecificExtraction> extractions) {
+                mExtractions = extractions;
+                if (mExtractions == null || mExtractions.isEmpty()) {
```

Comment:
I think a lot of the time we get some extractions, like `invoiceType` or similar even when there are no pay 5 extractions. You could use a method similar to `ExtractionsActivity#isPay5Extraction()` to check whether we got at least one pay 5 extraction.